models/public/mosaic/model.yml

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+    The `mosaic` model is one of the style transfer models designed to mix the content of 
+    an image with the style of another image. The model uses the method described in 
+    Perceptual Losses for Real-Time Style Transfer and Super-Resolution <https://arxiv.org/abs/1603.08155> along with 
+    Instance Normalization <https://arxiv.org/pdf/1607.08022.pdf>.
+task_type: "style_transfer"
+files:
+  - name: mosaic.onnx
+    sha256: fa646dedade881243f8d5a2ceb7de2b93675b21fc24f7482894ac4851a9a0a47
+    source: https://raw.githubusercontent.com/itlab-vision/OpenVINO-GAN-DEMO/master/sources/models/Mosaic/mosaic.onnx
+    size: 6728029
+
+model_optimizer_args:
+  - --input_shape=[1,3,224,224]
+  - --input_model=$dl_dir/mosaic.onnx
+  - --reverse_input_channels
+framework: onnx
+license: https://raw.githubusercontent.com/itlab-vision/OpenVINO-GAN-DEMO/master/LICENSE
```

Comment:
This isn't the same license as in the description file.

Reply:
License changed to the original repository license